Fluid/Values/FluidValue.cs

```diff
@@ -167,7 +167,7 @@ public static FluidValue Create(object value, TemplateOptions options)
                             return new DateTimeValue(dateTimeOffset);
 
                         case TimeSpan timeSpan:
-                            return new DateTimeValue(new DateTime(timeSpan.Ticks));
+                            return NumberValue.Create((decimal)timeSpan.TotalSeconds);
```

Comment:
Now I remember the difference. When I added `TimeSpan` support I intended it to be converted to a `DateTimeValue`. I need to check the issue that triggered it.

But this doesn't change the question about using the current TZ or not. I will take some more time to understand the intent in shopify. TimeSpan and Number my end up having different behavior in the end.

Reply:
Looking at Shopify's portal, when passing `now | date` or `0 | date` it's using the local TZ to create the date value. So I think we should do the same. I will update your PR to reflect this. I still want a TimeSpan to be converted to a date value though.